diff --git a/heifio/CMakeLists.txt b/heifio/CMakeLists.txt
index 920eeca5..663b5fdb 100644
--- a/heifio/CMakeLists.txt
+++ b/heifio/CMakeLists.txt
@@ -25,6 +25,7 @@ target_compile_definitions(heifio
         LIBHEIF_EXPORTS
         HAVE_VISIBILITY)
 
+find_package(WebP)
 find_package(TIFF)
 if (TIFF_FOUND)
     target_sources(heifio PRIVATE decoder_tiff.cc decoder_tiff.h encoder_tiff.h encoder_tiff.cc)
